cmd/admin_cmd.go

```diff
@@ -0,0 +1,161 @@
+package cmd
+
+import (
+	"github.com/Sirupsen/logrus"
+	"github.com/netlify/gotrue/conf"
+	"github.com/netlify/gotrue/models"
+	"github.com/netlify/gotrue/storage/dial"
+	"github.com/spf13/cobra"
+)
+
+var autoconfirm, isSuperAdmin, isAdmin bool
+var audience string
+
+func getAudience(c *conf.Configuration) string {
+	if audience == "" {
+		return c.JWT.Aud
+	}
+
+	return audience
+}
+
+func init() {
```

Comment:
personally, I prefer not using `init()`, I would rather have a method `AdminCmd() *cobra.Command` that did all of this and then had a locally scoped `adminCmd`. 

Feel free to ignore.